### ISSUE TITLE
Migration from "One Click Search" to 'expression-search'

### ISSUE THREAD

**Issue by taurus-forever** [wangvisual/expression-search#72]
Hi,

After upgrade to Thunderbird 60, the addon "One Click Search" has stopped working completely :-( 
So I am considering to migrate on your's 'expression-search'. My Thunderbird version is 60.2.1 (64-bit). Linux Mint 19.

I have installed beta version of the plugin https://github.com/wangvisual/expression-search/blob/master/beta/gmailui-1.4-tb.xpi
and enabled Crtl+RClick search in settings, so far nothing happens when I am trying to perform the Ctrl+Rclick on a email subject in the mail list pane. I have also tried Shift+RClick.

Also I am curious, is it possible to add support of Middle button there?
"One Click Search" supported it and it was VERY useful to click on mouse wheel instead of pressing Crtl/Shift, etc.

I have enabled "Show verbose information in error console", so far the most of the time nothing there at all, once I saw:
> window.controllers/Controllers is deprecated. Do not use it for UA detection.

While it might be unrelated.

Tnx! Waiting for your reply here.

**Comment by wangvisual** [comment]
that beta version missing one file

**Comment by wangvisual** [comment]
you need to download the new beta

**Comment by taurus-forever** [comment]
I have downloaded new https://github.com/wangvisual/expression-search/blob/master/beta/gmailui-1.4-tb.xpi and installed it.
I was not able to make it work at all. Nothing happens when I am clicking Ctrl+RClick or Shift+RClick
(just default Thunderbird actions happen for selecting messages, no quick search involved at all).
Maybe some garbage in my old Thunderbird... I have stopped trying. JFYI here only.

Tnx anyway!
